src/main/java/mchorse/blockbuster/client/gui/dashboard/panels/snowstorm/sections/GuiSnowstormGeneralSection.java

```diff
@@ -66,6 +67,17 @@ public GuiSnowstormGeneralSection(Minecraft mc, GuiSnowstorm parent)
 
         this.fields.add(this.identifier, Elements.row(mc, 5, 0, 20, this.pick, this.material));
     }
+    
+    @Override
+    protected void collapseState()
+    {
+        GuiSectionManager.setDefaultState(this.getClass().getSimpleName(), false);
+        
+        if (!GuiSectionManager.isCollapsed(this.getClass().getSimpleName()))
```

Comment:
Instead of copy-pasting, you should rather `super.collapseState()`. That also relates to other methods. Please fix.